ivi-layermanagement-api/ilmControl/src/ilm_control_wayland_platform.c

```diff
@@ -2140,18 +2140,20 @@ static void screenshot_done(void *data, struct ivi_screenshot *ivi_screenshot,
 
     if ((filename_ext = strstr(filename, ".png")) && (strlen(filename_ext) == 4)) {
         if (save_as_png(filename, (const char *)image_buffer,
-                              width, height, bytes_per_pixel * 8, has_alpha) == 0) {
+                        width, height, bytes_per_pixel * 8, has_alpha) == 0) {
             ctx_scrshot->result = ILM_SUCCESS;
         } else {
             fprintf(stderr, "failed to write screenshot as png file: %m\n");
```

Comment:
@rbmeftah Looks good, thank you. One thing which should be checked is whether `ctx_scrshot->result` shall be set to ILM_FAILED if the function fails to write a screenshot. If this variable is allocated somewhere on stack, it could have a random value. Maybe @efriedrich has additional comments.

Reply:
ctx_scrshot->result is initialized to ILM_FAILED